gateway/ln-gateway/src/lib.rs

```diff
@@ -58,6 +58,10 @@ const ROUTE_HINT_RETRY_SLEEP: Duration = Duration::from_secs(2);
 /// LND HTLC interceptor can't handle SCID of 0, so start from 1
 const INITIAL_SCID: u64 = 1;
 
+/// Liquidity-based routing fee in millionths of a routed amount.
+/// The default is 10000 (1%). In other words, 10000 is 1%.
+pub const DEFAULT_FEES_PROPORTIONAL_MILLIONTHS: u32 = 10000;
```

Comment:
Could be a `Default` on `FeeStructure`

Reply:
Reused extern definition of [RoutingFees](https://github.com/lightningdevkit/rust-lightning/blob/main/lightning/src/routing/gossip.rs#L1057-L1065) so we actually need a default const